cmd/utils/flags.go

```diff
@@ -568,11 +568,6 @@ var (
 		Value:    ethconfig.Defaults.Miner.Recommit,
 		Category: flags.MinerCategory,
 	}
-	MinerNoVerifyFlag = &cli.BoolFlag{
```

Comment:
What if we keep this flag as well for ethash?

Reply:
SGTM, I removed it in a flurry.

Reply:
[2ddfa04](https://github.com/etclabscore/ethereum.go-ethereum/pull/18/commits/2ddfa042b2de815273061ba254390e4eda4029fe) re-installs it.